### PR TITLE
Scripts/Exile's Reach: Ship both factions

### PR DESCRIPTION
**Pull request by ghost**
Scripting to get Alliance and Horde tunes from ship to the shore.

**Changes proposed:**

-  Add missing scripting to get player to shore. Will continue scripting quest on shore after this PR is added.

**Tests performed:**
Builds, tested both sides in game.

**Known issues and TODO list:** (add/remove lines as needed)

No known issues

### CODE REVIEW

File: sql/updates/world/master/Exiles_reach.sql [hunk 1]
```diff
@@ -0,0 +1,499 @@
...
+(128,0,@CGUID+21),
+(128,0,@CGUID+22),
+(128,0,@CGUID+23),
+(128,0,@CGUID+24),
+(128,0,@CGUID+25);
+
+-- *** Creature Fixes ***
+
+-- Phase updates
+UPDATE `creature` SET `PhaseId`=15286 WHERE `id` IN (166573,166824); -- Warlord Breka Grimaxe
```
Comment (Naddley) [inline sql/updates/world/master/Exiles_reach.sql:65]: There is no PhaseShift after complete the first Quest so they are not phased.

**Comment by malcrom** [comment]
Only two core issues left.

File: sql/updates/world/master/Exiles_reach.sql (hunk 1 above)
Comment (malcrom) [reply sql/updates/world/master/Exiles_reach.sql]: Changed

**Comment by malcrom** [comment]
This PR is complete.

**Pull request merged**